docs/tutorial/write-your-resolvers.md

```diff
@@ -0,0 +1,125 @@
+---
+id: write-your-resolvers
+title: Write your resolvers
+sidebar_label: 5. Write your resolvers
+---
+
+[Previsouly](/docs/tutorial/create-server), we created our application instance with a [first iteration of our Schema](/docs/tutorial/create-server#recipes-manager-sdl-querygraphql), based on the SDL. Aside, we are going to bind some datas to our Schema, thank to the resolvers, which are in charge of make the bridge between the Schema and your application logic. 
+
+## What is a resolver?
+
+A resolver is a function _(which could be asynchronous)_ ties to a field of your Schema. It will be in charge of providing data accordingly to the fields or objects it is attached on.
```

Comment:
is a function _(which can be asynchronous)_ which is tied to a field in your Schema.

attached to